portfolio/src/main/webapp/script.js

```diff
@@ -42,14 +42,14 @@ async function getGreetingUsingAsyncAwait() {
 function loadComments() {
   fetch('/data').then(response => response.json()).then((comments) => {
     const commentListElement = document.getElementById('comments-container');
-    comments.forEach((comments) => {
-      commentListElement.appendChild(createCommentElement(comments));
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createCommentElement(comment.name, comment.message, comment.time));
```

Comment:
This looks longer than 80 characters. Break the line to avoid that. You might break at `comment.name` and indent the rest of the line an additional 4 spaces.